LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Global Public Inclusive Infrastructure
```

Comment:
Yes, that's correct. The attribution should be to your employer; in this case, Raising the Floor - International.